src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php

```diff
@@ -297,7 +297,7 @@ public function enterNode(Node $node): ?int
                     // Some constants has different values on different systems, some are not actual in stubs
                     if (defined($constantName)) {
                         /** @var scalar|scalar[]|null $constantValue */
-                        $constantValue        = constant($constantName);
+                        $constantValue        = @constant($constantName);
```

Comment:
We should really not introduce a hard suppression here.

If it is required, we really need a comment around it

Reply:
https://github.com/kukulich/BetterReflection/commit/d93b342402bde05fc72bd2e85a8b833b9015ed60